packages/compose/index.ts

```diff
@@ -30,14 +30,14 @@ export type ComposableFactoryParams = Parameters<ComposableFactory>;
  * @param {...(Composable)} [arguments] The list of composables.
  * @returns {Stamp} A new stamp (aka composable factory function)
  */
-interface ComposeMethod {
+interface ComposeMethod extends PropertyMap {
```

Comment:
The `ComposeMethod` (actually we should some day rename it to `ComposeFunction`, the word `method` is wrong here)
is also in the specification: https://github.com/stampit-org/stamp-specification#standalone-compose-pure-function-optional
It does not have any properties on it.

Please, remove `PropertyMap` from both `ComposeMethod` here and `ComposeProperty` below.

Reply:
Can change it now. Will remove and report back

Reply:
Removed it and no other changes were necessary.